### PR TITLE
Release 0.4.0-alpha.1

### PR DESCRIPTION
**Pull request by rartych**
#### What type of PR is this?

Add one of the following kinds:
* subproject management

#### What this PR does / why we need it:

This is "Release PR: for Commonalities Release 0.4.0-alpha.1

#### Changelog input

```
 Commonalities Release 0.4.0-alpha.1
```

#### Additional documentation 

https://wiki.camaraproject.org/display/CAM/Meta-release+Process#MetareleaseProcess-Milestoneactivities

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,59 @@
...
+* Commonalities approved deliverables in **[documentation](https://github.com/camaraproject/Commonalities/tree/r0.4.0-alpha.1/documentation)** folder
+* Commonalities approved artifacts in **[artifacts](https://github.com/camaraproject/Commonalities/tree/r0.4.0-alpha.1/artifacts)** folder
+  - **New:**
+    - [Notification Subscription Template](https://github.com/camaraproject/Commonalities/blob/r0.4.0-alpha.1/artifacts/camara-cloudevents/event-subscription-template.yaml)
+
+### Added
+* Usage and style of operation tags in API Design Guidelines by @rartych in https://github.com/camaraproject/Commonalities/pull/152
+* x-correlator support in notifications in API Design Guidelines by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/170
+* Create subscription-notification-template.yaml by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/189
+
```
Comment (shilpa-padgaonkar) [inline CHANGELOG.md:31]:
```suggestion
* Added a different scope naming format for APIs that deal with explicit subscriptions by @shilpa-padgaonkar in https://github.com/camaraproject/Commonalities/pull/177 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,59 @@
...
+
+### Changed
+* API Design Guidelines updated with subscriptionMaxEvents by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/131
+* 'specversion' in CloudEvents as enum by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/187
+* 'datacontenttype' in CloudEvents as enum by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/193
+* API Design Guidelines updated on Explicit Subscriptions by @shilpa-padgaonkar in https://github.com/camaraproject/Commonalities/pull/177
```
Comment (shilpa-padgaonkar) [inline CHANGELOG.md:37]:
```suggestion

```

**Review by PedroDiez — APPROVED** [review]
LGTM so far

**Review by shilpa-padgaonkar — APPROVED** [review]
/LGTM

**Comment by rartych** [comment]
@patrice-conil Could you approve it?

**Review by patrice-conil — APPROVED** [review]
LGTM

**Pull request merged**